config/locales/en.yml

```diff
@@ -455,3 +455,13 @@ en-odr:
         A project you are managing has been closed by %{current_user}.
 
         Closure reason: %{closure_reason}
+    application_transitioned_to_contract_completed:
+      body_html: >
+        <p>A project you are managing has had the contract approved by %{current_user}.</p>
```

Comment:
I feel like wording should be 'application' but I guess it is 'project' on the acceptance criteria.